README.md

```diff
@@ -128,6 +128,22 @@ This is an active open-source project. We are always open to people who want to
 Contact us if you are looking for implementation tasks that fit your skills.
 This article describes [how to contribute to Apache Flink](https://flink.apache.org/contributing/how-to-contribute.html).
 
+## Externalized Connectors
+
+Most Flink connectors have been externalized to individual repos under the [Apache Software Foundation](https://github.com/apache):
+
+* [flink-connector-aws](https://github.com/apache/flink-connector-aws)
+* [flink-connector-cassandra](https://github.com/apache/flink-connector-cassandra)
+* [flink-connector-elasticsearch](https://github.com/apache/flink-connector-elasticsearch)
+* [flink-connector-gcp-pubsub](https://github.com/apache/flink-connector-gcp-pubsub)
+* [flink-connector-hbase](https://github.com/apache/flink-connector-hbase)
+* [flink-connector-jdbc](https://github.com/apache/flink-connector-jdbc)
+* [flink-connector-kafka](https://github.com/apache/flink-connector-kafka)
+* [flink-connector-mongodb](https://github.com/apache/flink-connector-mongodb)
+* [flink-connector-opensearch](https://github.com/apache/flink-connector-opensearch)
+* [flink-connector-prometheus](https://github.com/apache/flink-connector-prometheus)
```

Comment:
sorry didn't look to every at the first time
it seems there is the same issue with Prometheus connector FLINK-33137
it is not implemented yet... I opened the link to the connector and there is an empty repo with basic setup

Reply:
No worries, many thanks for the checking them! I think this one is a little bit different, since it was never in Flink repo. It is a new connector under development now. Therefore it should be fine to show the lin. WDYT?

Reply:
I'm not against it
the only confusing thing for me is that i try to look here from user's point of view who in theiry expects something working if it is mentioned at website...and for this connector there is nothing so far since the work is in progress

All other thinks are ok :+1:  I would leave the decision here up to you since i don't have a strong opinion about that

Reply:
Thanks!